admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -47,4 +47,11 @@ export default {
   showOrderShippingUpdateModalBtn: '.js-update-shipping-btn',
   updateOrderShippingTrackingNumberInput: '#update_order_shipping_tracking_number',
   updateOrderShippingCurrentOrderCarrierIdInput: '#update_order_shipping_current_order_carrier_id',
+  orderMessageNameSelect: '#order_message_order_message',
+  orderMessagesContainer: '.js-order-messages-container',
+  orderMessage: '#order_message_message',
+  orderMessageChangeWarning: '.js-message-change-warning',
+  allMessagesModal: '#viewAllMessagesModal',
```

Comment:
Why camelCalse here and snake-case below ?

Reply:
sorry, probably copied from somewhere - will replace it  with underscores due to I saw that all ids are snake_case